packages/client/src/utils/rxjs/BroadbandObservable.ts

```diff
@@ -0,0 +1,24 @@
+import { Observable } from 'rxjs';
+import type { StreamrClient } from 'streamr-client';
+import { MessageListener, Stream } from 'streamr-client';
+import { MessageMetadata } from 'streamr-client/types/src/Message';
+
+import { BroadbandSubscriber } from '../BroadbandSubscriber';
+
+export const createBroadbandObservable = (
```

Comment:
A comment on this would also be nice.
The comments elsewhere are awesome!

Reply:
Done!